config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/BuilderConstants.java

```diff
@@ -0,0 +1,4 @@
+package com.ca.apim.gateway.cagatewayconfig.bundle.builder;
+
+public class BuilderConstants {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this empty class, write its code or make it an "interface". ([squid:S2094](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS2094))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=138&issues=AWgqdHfASi5GWQS9qtJG&open=AWgqdHfASi5GWQS9qtJG)

[ISSUE_KEY]: AWgqdHfASi5GWQS9qtJG